documentation/book/assembly-storage.adoc

```diff
@@ -8,9 +8,13 @@
 :parent-context: {context}
 
 [id='assembly-storage-{context}']
-= Kafka and Zookeeper storage
+= Kafka and Zookeeper storage types
 
-As stateful applications, Kafka and Zookeeper need to store data on disk. {ProductName} supports three different types of storage for this data: ephemeral, persistent, and JBOD storage.
+As stateful applications, Kafka and Zookeeper need to store data on disk. {ProductName} supports three storage types for this data:
+
+* Ephemeral
+* Persistent
+* JBOD storage.
```

Comment:
If you're going to have a period here then the three before should be formatted as if they were in a sentence - like

* Ephemeral,
* Persistent,
* and JBOD storage.

Reply:
Will remove it.